src/processTargets/getModifierStage.ts

```diff
@@ -5,6 +5,11 @@ import {
   Modifier,
 } from "../typings/targetDescriptor.types";
 import CascadingStage from "./modifiers/CascadingStage";
+import {
+  ContentStage,
+  EmptyStage,
+  WhitespaceStage,
```

Comment:
I wonder if we should be a bit more verbose, and call these eg `ContentFilterStage`, `EmptyFilterStage`, and `WhitespaceFilterStage`.  Ie all filter stages are suffixed with `FilterStage`

Reply:
Done. Should we still keep the modifier type eg "content" or do you prefer "contentFilter"?